.github/workflows/platform-executables-acceptance-tests.yml

```diff
@@ -52,61 +57,60 @@ jobs:
   get-tag:
     runs-on: ubuntu-latest
     outputs:
-     TAG_NAME: ${{steps.get-tag.outputs.TAG_NAME}}
+      TAG_NAME: ${{steps.get-tag.outputs.TAG_NAME}}
     steps:
-     - uses: actions/checkout@v2
-     - name: Getting latest tag
-       id: get-tag
-       run: |
-        git fetch --prune --unshallow
-        echo "::set-output name=TAG_NAME::$(git describe --tags $(git rev-list --tags --max-count=1))"
+      - uses: actions/checkout@v2
+      - name: Getting latest tag
+        id: get-tag
+        run: |
+          git fetch --prune --unshallow
+          echo "::set-output name=TAG_NAME::$(git describe --tags $(git rev-list --tags --max-count=1))"
   platform-exec-test:
     name: Test for ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     needs: [get-tag, wait-for-releases]
     strategy:
-     fail-fast: false
-     matrix:
+      fail-fast: false
+      matrix:
         include:
           - os: ubuntu-latest
-            extension_type: '.deb'
+            extension_type: ".deb"
             cmds_to_execute: sudo dpkg -i twilio-${{ needs.get-tag.outputs.TAG_NAME }}.deb
           - os: macos-latest
-            extension_type: '.pkg'
+            extension_type: ".pkg"
```

Comment:
[Q] Why do we need this change? 

Seems harmless, but we could avoid pushing without testing `' '` v/s `" "`

Reply:
It's my vscode being extra, reverted them